getblock.go

```diff
@@ -106,6 +106,9 @@ func getHash(height int64) (hash string, err error) {
 }
 
 func blockFromBlockchainInfo(hash string) (string, error) {
+	if network != "main" {
+		return "", fmt.Errorf("%s is not available for blockchain.info", network)
+	}
```

Comment:
Instead of crashing you could just not use blockchain.info when in testnet, use just blockchair.

Reply:
Right. Crashing is not good.

Reply:
But it will just cointinue to the next block provider?!

Reply:
No, it uses two functions interchangeably, they're defined here: https://github.com/fiatjaf/trustedcoin/blob/16b5c9658c59f587377ce38a8586bb806bbc023b/getblock.go#L27-L30

That is a list of functions (`[]func(string) (string, error){}`). You must define that previously instead of inline, and do it with only blockchair for testnet, both for mainnet.